packages/api-client/index.html

```diff
@@ -2,141 +2,16 @@
 
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8" />
-    <title>Demo</title>
-    <link rel="shortcut icon" href="data:image/x-icon;" type="image/x-icon" />
-    <style>
-      .login-page {
-        width: 360px;
-        padding: 8% 0 0;
-        margin: auto;
-      }
 
-      .form {
-        position: relative;
-        z-index: 1;
-        background: #FFFFFF;
-        max-width: 360px;
-        margin: 0 auto 100px;
-        padding: 45px;
-        text-align: center;
-        box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.2), 0 5px 5px 0 rgba(0, 0, 0, 0.24);
-      }
+<head>
+  <meta charset="UTF-8" />
+  <title>Demo</title>
+  <link rel="shortcut icon" href="data:image/x-icon;" type="image/x-icon" />
+  <script src="./dist/demo.js"></script>
+</head>
 
-      .form input {
-        font-family: "Roboto", sans-serif;
-        outline: 0;
-        background: #f2f2f2;
-        width: 100%;
-        border: 0;
-        margin: 0 0 15px;
-        padding: 15px;
-        box-sizing: border-box;
-        font-size: 14px;
-      }
+<body>
+  <div id="app"></div>
+</body>
```

Comment:
Indent `<body>` and `<head>`?

Reply:
I don't think we need to, as you see them not indented in lots of places [citation needed]. The only parent element will always be `<html>` so there is no real gain in indenting them.

Reply:
My OCD... but I can see that most of the formatters are doing that.. :) So let's keep it..